### ISSUE TITLE
Variables window for array doesn't list elements in numerical order

### ISSUE THREAD

**Issue by beltaurus** [richterger/Perl-LanguageServer#18]
When debugging a script with the following statement:
```
my %hash = (
    'a'  => 1,
    'b'  => 2,
    '_c' => 3,
    'z'  => 4,
    'k'  => 5,
    'aa' => 6,
);

my @hash_copy = %hash;
```

And running the debugger, the following shows up for the hash and the array:
![image](https://user-images.githubusercontent.com/54114043/79412363-256e6f00-7f5a-11ea-8276-831612c552d9.png)

For the array `@hash_copy`, I am expecting the indices to be in numerical order, yet they seem to be in alphanumerical order?

**Comment by beltaurus** [comment]
Actually, we don't even need to recast a hash into an array: simply try:
```
my @array = ( 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0 );
```
And the result is also "scrambled" in the debugger:
![image](https://user-images.githubusercontent.com/54114043/79462946-0e0ca180-7fad-11ea-836e-0479594e942b.png)

**Comment by richterger** [comment]
Fixed in commit 76a8681fdd9905467539764492a9b4d547a8ff41
